chat/input-bar/src/InputBar/InputBarFeedback.tsx

```diff
@@ -52,7 +52,7 @@ export const InputBarFeedback = forwardRef<
           <div className={loadingContainerStyles}>
             <AssistantAvatar darkMode={darkMode} size={20} />
             <Body className={getLoadingTextStyles(theme)}>
-              {loadingMessage || messages.loading}
+              {`${assistantName} is thinking`}
```

Comment:
Will folks still want to customize the "is thinking" part of the string? 

OTOH, this locks down the copy, which might be a good thing